roo.gemspec

```diff
@@ -6,21 +6,20 @@ require 'roo/version'
 Gem::Specification.new do |spec|
   spec.name          = 'roo'
   spec.version       = Roo::VERSION
-  spec.authors       = ['Thomas Preymesser', 'Hugh McGowan', 'Ben Woosley']
-  spec.email         = ['ruby.ruby.ruby.roo@gmail.com']
+  spec.authors       = ['Thomas Preymesser', 'Hugh McGowan', 'Ben Woosley', 'Oleksandr Simonov']
+  spec.email         = ['ruby.ruby.ruby.roo@gmail.com', 'oleksandr@simonov.me']
   spec.summary       = 'Roo can access the contents of various spreadsheet files.'
   spec.description   = "Roo can access the contents of various spreadsheet files. It can handle\n* OpenOffice\n* Excelx\n* LibreOffice\n* CSV"
   spec.homepage      = 'http://github.com/roo-rb/roo'
   spec.license       = 'MIT'
 
   spec.files         = `git ls-files -z`.split("\x0")
-  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
-  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
+  spec.files.reject! { |fn| fn.include?('test/files') }
```

Comment:
Specs and tests are included in the `git ls-files -z` command

    $ ruby -e 'p `git ls-files -z`.split("\x0").select { |f| f.include? "spec.rb" }.size'
    # => 8